packages/ui-utils/src/parse-complex-response.ts

```diff
@@ -116,6 +127,7 @@ export async function parseComplexResponse({
           role: 'assistant',
           content: '',
           createdAt,
+          parts: [],
```

Comment:
if we make parts optional this might not be necessary unless the user provides files